advanced/bash-completion/pihole

```diff
@@ -41,23 +37,7 @@ _pihole() {
 		"version")
 			opts_version="\--admin \--current \--ftl \--hash \--latest \--pihole"
```

Comment:
This is a suggestion, but it will also need a change in `version.sh`:

The name "adminLTE" was replaced by "web", but the option (in version.sh) is still using `-a` and `admin`.

We could replace them with `-w` and `web`.

What do you think?

Reply:
Good idea!